### PR TITLE
Preparation for the Cockpit release v1.0.0

### PR DESCRIPTION
**Pull request by Mishytca**

### CODE REVIEW

**Comment by rshym** [comment]
It looks like it's better to squash both commits into one "Fixing all components", considering that SRCREV for `u-boot-android` is fixed in the first commit and the second commit fixes SRCREV for "all" components.

Also I see that `meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/linux/linux-renesas_%.bbappend` is missed and still has `SRCREV = "${AUTOREV}"`.

File: prod-cockpit-rcar.yaml [hunk 1]
```diff
@@ -36,13 +36,13 @@ common_data:
...
-      rev: be26ff13057f8aaa503db8ef4f3826a4b483f35b
+      rev: dunfell-v0.1.0
     - type: git
       url: "https://github.com/xen-troops/meta-xt-rcar.git"
-      rev: da9485c357b041f5af6232e0a7f24bbbb3d72d56
+      rev: dunfell-v0.1.0
     - type: git
-      url: "https://github.com/xen-troops/meta-xt-prod-cockpit-rcar.git"
-      rev: master
+      url: "https://github.com/Mishytca/meta-xt-prod-cockpit-rcar.git"
```
Comment (rshym) [inline prod-cockpit-rcar.yaml:44]: Please replace `Mishytca` with `xen-troops`. We should not use developer's forks in PRs.

File: prod-cockpit-rcar.yaml [hunk 2]
```diff
@@ -36,13 +36,13 @@ common_data:
...
+      rev: release-v1
```
Comment (rshym) [inline prod-cockpit-rcar.yaml:45]: As we discussed, please use the release tag `v1.0.0` that will be created after the merge of this PR.

Comment (Mishytca) [reply prod-cockpit-rcar.yaml]: Hi @rshym, done.

**Review by rshym — APPROVED** [review]
`Reviewed-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>`

**Comment by Mishytca** [comment]
Rb is added. Successfully built and tested on h3ulcb-4x2g-kf.
Ready to be merged.

**Pull request merged**